config/config.toml

```diff
@@ -18,7 +18,7 @@ constants = "minimal"
 active = false
 genesisTime = 1567777777
 ownedValidatorStartIndex = 0
-ownedValidatorCount = 16
+ownedValidatorCount = 8
```

Comment:
Let's leave the config.toml untouched unless you have a real reason to modify it